OpenSim/Simulation/Model/FunctionBasedBushingForce.cpp

```diff
@@ -186,6 +186,7 @@ SimTK::Vec6 FunctionBasedBushingForce::
     calcDampingForce(const SimTK::State& s) const
 {
     Vec6 dqdot = computeDeflectionRate(s);
+    // This _dampingMatrix is initialized to zero and never updated!
```

Comment:
@aseth1 Where should this _dampingMatrix be computed/updated?  Is finalizeFromProperties() the correct place? Thanks.